pwndbg/disasm/__init__.py

```diff
@@ -258,86 +284,91 @@ def can_run_first_emulate() -> bool:
 first_time_emulate = True
 
 
-def near(address, instructions=1, emulate=False, show_prev_insns=True):
+# Return (list of PwndbgInstructions, index in list where instruction.address = passed in address)
+def near(
+    address, instructions=1, emulate=False, show_prev_insns=True, use_cache=False
+) -> tuple[list[PwndbgInstruction], int]:
```

Comment:
I think for Python 3.8 we need to use `Tuple` and `List` from `typing`.

Reply:
I think we could probably still use this kind of annotation with `from __future__ import annotations`?
Or will this `from __future__` cause any side effects?

Reply:
I would need to test it, but I thought that the `annotations` import would only allow for using the postponed evaluation feature in https://peps.python.org/pep-0563/.